bindings/pydeck/examples/hexagon_layer.py

```diff
@@ -0,0 +1,30 @@
+import pydeck as pdk
+
+HEXAGON_LAYER_DATA = ('https://raw.githubusercontent.com/uber-common/'
+                      'deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv')
+
+# Define a layer to display on a map
+layer = pdk.Layer(
+    'HexagonLayer',
+    HEXAGON_LAYER_DATA,
+    get_position=['lng', 'lat'],
```

Comment:
@ajduberstein Does this work, can we send in Python arrays as expressions without quotes? Is this documented?

Reply:
Yes, it works–the documentation is made explicit in the [latest docs](https://github.com/uber/deck.gl/pull/4270/files#diff-839498afeb7ab08f9dba4e095fab6d24R183).